tree/ntuple/v7/inc/ROOT/RNTupleModel.hxx

```diff
@@ -307,37 +326,67 @@ public:
    /// Creating projections for fields containing `std::variant` or fixed-size arrays is unsupported.
    RResult<void> AddProjectedField(std::unique_ptr<ROOT::RFieldBase> field, FieldMappingFunc_t mapping);
 
+   /// Transitions a RNTupleModel from the Building state to the Frozen state, disabling adding additional fields and enabling
```

Comment:
```suggestion
   /// Transitions an RNTupleModel from the *building* state to the *frozen* state, disabling adding additional fields and enabling
```
same applies below

Reply:
Whereas I can agree with the emphasis/non-capitalization in the class comment (as it's the first time the states are introduced), I'd argue that here the capitalization fits better because we are referring to a precise, named concept (corresponding to the nodes of the ascii graph above and to the values of the EState enum - which, sure, are private, but are still visible in the documentation)

Reply:
Ok fair enough, I'm personally not a big fan of lots of capitalization but I won't stay awake over it :)

Reply:
Just one thing I noticed now, in line 114-116 is is written italicized and uncapitalized